missilewars-plugin/src/main/java/de/butzlabben/missilewars/game/GameManager.java

```diff
@@ -133,6 +165,12 @@ public Game getGame(String name) {
         return games.get(name);
     }
 
+    public void addGame(String name, Game game) {
+        games.put(name, game);
+        List<String> gameNames = new ArrayList<>(games.keySet());
+        MissileWars.getInstance().commandManager.getCommandCompletions().registerCompletion("games", c -> gameNames);
```

Comment:
**What I changed:**
This line adds the `@games` tab-completion with the list of the games for commands.